tjreports/site/views/reports/tmpl/default.php

```diff
@@ -313,7 +335,7 @@ function getQueryResult(id, Itemid)
 
 function loadReport(reportToLoad,mid)
```

Comment:
@hemantpurohit1  Move this to JS file. If you do not have js file create one and include it on this view.